### PR TITLE
Tardigrade documentation

### PR DESCRIPTION
**Pull request by kaloyan-raev**
#### What is the purpose of this change?

Adds documentation for the Tardigrade backend.

Also moves the backend implementation from `backend/storj` to `backend/tardigrade` to resolve a build failure with `make doc`.

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/tardigrade.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+
+    rclone delete remote:bucket/path/to/dir/
+
+### Print the total size of objects
+
+Use the `size` command to print the total size of objects in a bucket or a folder.
+
+    rclone size remote:bucket/path/to/dir/
+
+Sync `/home/local/directory` to the remote bucket, deleting any excess
```
Comment (mniewrzal) [inline docs/content/tardigrade.md]: This sentence doesn't fit `Print the total size of objects` paragraph.

Comment (kaloyan-raev) [reply docs/content/tardigrade.md]: True. Forgot to remove it. Removed now.

**Comment by mniewrzal** [comment]
lgtm

File: docs/layouts/chrome/navbar.html [hunk 2]
```diff
@@ -88,6 +88,7 @@
                     <li><a href="/putio/"><i class="fas fa-parking"></i> put.io</a></li>
                     <li><a href="/sftp/"><i class="fa fa-server"></i> SFTP</a></li>
                     <li><a href="/sugarsync/"><i class="fas fa-dove"></i> SugarSync</a></li>
+                    <li><a href="/storj/"><i class="fas fa-dove"></i> Tardigrade</a></li>
```
Comment (calebcase) [inline docs/layouts/chrome/navbar.html]: Should this be `/tardigrade/`?

Comment (kaloyan-raev) [reply docs/layouts/chrome/navbar.html]: Yes. Fixed.

**Review by calebcase — APPROVED** [review]

**Pull request merged**